lib/Jsrt/Jsrt.cpp

```diff
@@ -1439,7 +1399,7 @@ CHAKRA_API JsDefineProperty(_In_ JsValueRef object, _In_ JsPropertyIdRef propert
 
 CHAKRA_API JsCreateArray(_In_ unsigned int length, _Out_ JsValueRef *result)
 {
-    return ContextAPIWrapper<true>([&] (Js::ScriptContext *scriptContext) -> JsErrorCode {
+    return ContextAPINoScriptWrapper([&](Js::ScriptContext *scriptContext) -> JsErrorCode {
```

Comment:
>ContextAPINoScriptWrapper([&](J [](start = 11, length = 31)

Some changes unrelated to this PR? Need to evaluate carefully if these are safe.